src/utils.ts

```diff
@@ -1,5 +1,13 @@
+import bignum from 'bignum';
+import { ConfigManager } from 'confmgr/lib';
 import log4js from 'log4js';
 
+import { faucetConfig } from './faucetConfig';
+
+const config = faucetConfig('server');
+const decimals = config.Get('BACKEND', 'NETWORK_DECIMALS') as number;
+const MAX_ALLOWED_DRIP_FLOAT = 9999999.9;
```

Comment:
Why can't this variable be named `maxAllowedDripFloat`? It's inconsistent like this

Reply:
Got it. I mean the naming convention is inconsistent with other constants. For me it's not a big deal to leave this as it is.

Reply:
I see what you mean. What you tell is `const DECIMALS` and other similar would need to look also UpperSnakecase to be consistent? ) yea, that's true

Reply:
> other similar would need to look also UpperSnakecase to be consistent

I would *prefer* the other way around: all variables should be `camelCase`. Regardless, I think consistency is what's important, regardless of preference.